### PR TITLE
Merge promq code into instrumentation-tools

### PR DESCRIPTION
**Pull request by logicalhan**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

Merges promq into SIG-repo proper

### CODE REVIEW

**Comment by RainbowMango** [comment]
Seems some author didn't sign CLA.

promq code? Where is these code porting from?

**Comment by lilic** [comment]
Very nice, I have been looking forward to promq since the demo and used it myself, thanks for the contribution! 🎉 lgtm from my side!

File: cmd/metrics/data.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2020 Google LLC
```
Comment (lilic) [inline cmd/metrics/data.go]: Does this copyright stay to google?

Comment (logicalhan) [reply cmd/metrics/data.go]: Updated, sorry that was auto-generated at one point.

**Comment by lilic** [comment]
I wonder if we want to call this repo instrumentation-tools I would expect for this repo to contain multiple tools, so promq being the start. If that is the case, then preferably we should have this be self-contained in a separate directory (e.g. we can make it submodule for example).

When I was trying it out, the README was not correct, as the actual `promq.go` was one dir level above https://github.com/kubernetes-sigs/instrumentation-tools/tree/6cf294968c2666c4812e3baf2520450215910e10/promq

**Comment by logicalhan** [comment]
> I wonder if we want to call this repo instrumentation-tools I would expect for this repo to contain multiple tools, so promq being the start. If that is the case, then preferably we should have this be self-contained in a separate directory (e.g. we can make it submodule for example).
> 
> When I was trying it out, the README was not correct, as the actual `promq.go` was one dir level above https://github.com/kubernetes-sigs/instrumentation-tools/tree/6cf294968c2666c4812e3baf2520450215910e10/promq

Ah yeah, that's not super surprising to me, we modifying the directory structure a bit so things may be inaccurate. Would it be cool if we merge this and then we can update the readme in a followup PR?

I would love to get this merged so that we can iterate over it in the SIG repo.

**Comment by lilic** [comment]
Happy from my side to get this merged, I am not in the OWNERS file so can't lgtm :), but no blockers from my side, its a cool tool! 👍  Just wanted to clear up that this is a repo we can use for other similar tools in the future?

**Comment by lilic** [comment]
/lgtm

🎉

**Pull request merged**